### ISSUE TITLE
Typhoeus/Parallel request Compatability 

### ISSUE THREAD

**Issue by jphenow** [remiprev/her#74]
# Background

Hey, awesome gem! I've been working on something of an ORM myself, and I wanted to implement parallel requesting with [Typhoeus](https://github.com/typhoeus/typhoeus). I might still implement this myself, but I really enjoy this implementation and figured this would be a good spot to add such functionality.
# The Question

So, does anyone have any interest in cooperating to add parallel requesting with Typhoeus on here?
# Initial glance

At a glance, just using:

``` ruby
Her::API.setup :url => "http://example-url.example.com" do |connection|
  # ...
  connection.use Faraday::Adapter::Typhoeus
end
```

There's an error upon request:

``` bash
Ethon::Errors::InvalidOption: The option: disable_ssl_peer_verification is invalid.
from ...@broker/gems/ethon-0.5.7/lib/ethon/easy.rb:254:in `block in set_attributes'
```
# tl;dr

Hoping to get some parallel request implementation, potentially on Typhoeus, what are your thoughts?
